dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -69,6 +69,57 @@ def test_profiled_precision(self):
         float_profiler.update(df_mix)
         self.assertEqual(1, float_profiler.precision)
 
+        # edge cases #
+        # integer with 0s on right and left side
+        df_ints = pd.Series(['0013245678', '123456700', '0012345600'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_ints)
+        self.assertEqual(6, float_profiler.precision)
+
+        # scientific
+        df_scientific = pd.Series(['1.23e-3', '2.2344', '1.244e4'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_scientific)
+        self.assertEqual(3, float_profiler.precision)
+
+        # plus
+        df_plus = pd.Series(['+1.3e-3', '+2.244', '+1.3324e4'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_plus)
+        self.assertEqual(2, float_profiler.precision)
+
+        # minus
+        df_minus = pd.Series(['-1.3234e-3', '-0.244', '-1.3324e4'])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_minus)
+        self.assertEqual(3, float_profiler.precision)
+
+        # spaces around values
+        df_spaces = pd.Series(['  -1.3234e-3  ', '  -0.244  '])
+        float_profiler = FloatColumn("Name")
+        float_profiler.update(df_spaces)
+        self.assertEqual(3, float_profiler.precision)
+
+        # check to make sure all formats of precision are correctly predicted
+        samples = [
+            # value, expected precision
+            ['10.01', 4],
+            ['.01', 1],
+            ['0.01', 1],
+            ['-0.01', 1],
+            ['+0.01', 1],
+            [' +0.013', 2],
+            ['  -1.3234e-3  ', 5],
+            ['  0012345600  ', 6],
+            ['  0012345600.  ', 8],
+            ['  -0012345600.  ', 8],
```

Comment:
Should toss a full nan's and none's in here.

Reply:
as above